lib/utils.c

```diff
@@ -849,3 +852,30 @@ sr_demangle_symbol(const char *sym)
 
     return demangled;
 }
+
+char*
+anonymize_path(char *orig_path)
+{
+    if (!orig_path)
+        return orig_path;
+    char* new_path = orig_path;
+    if (strncmp(orig_path, "/home/", strlen("/home/")) == 0)
+    {
+        // Find how rest after /home/.../
+        for (int i = 0; i < 3; i++)
+        {
```

Comment:
In this if branch you know `orig_path` starts with "/home/" because of `strcmp(...)`. You can use just, for instance:
```
new_path = strchr(new_path + strlen("/home/"), '/');
```